packages/node/package.json

```diff
@@ -24,11 +24,11 @@
     "@ud-viz/shared": "3.0.0",
     "child-process-promise": "^2.2.1",
     "express": "^4.18.2",
+    "gm": "^1.23.1",
+    "pngjs": "^6.0.0",
     "puppeteer": "^19.4.1",
     "socket.io": "^4.5.4",
-    "string-replace-middleware": "^1.0.2",
-    "gm": "^1.23.1",
-    "pngjs": "^6.0.0"
+    "string-replace-middleware": "^1.0.2"
```

Comment:
string replace should not be a depency of node package since it's only use in the mono repo context